pkg/component/component.go

```diff
@@ -244,7 +263,8 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 	}()
 
 	err = client.ExecCMDInContainer(pod.Name,
-		[]string{"/opt/app-root/bin/assemble-and-restart.sh"},
+		// We will use the assemble-and-restart script located within the supervisord container we've created
```

Comment:
`assemble-and-restart binary`

Reply:
why binary?

Reply:
oh sorry I thought this is binary, my bad